src/components/inputs/YearsCounter/YearsCounter.tsx

```diff
@@ -0,0 +1,53 @@
+import { useEffect, useState } from 'react';
+
+import './styles.css';
+
+function YearsCounter({ setCount, count }: { setCount: any; count: number }) {
+  const [years, setYears] = useState('year');
+  const [registration, setRegistration] = useState('');
+
+  useEffect(() => {
+    changeYear();
+  }, [count]);
+
+  function subCount() {
+    if (count > 1) {
+      setCount(count - 1);
+    } else {
+      return;
+    }
+  }
+  function addCount() {
+    setCount(count + 1);
```

Comment:
the contract will probably define this, but we should have a max-count for years. at least for now. maybe 5/10? what do you think?

Reply:
Actually I had the same question, currently the opinion from the others was no max count. Ill ping in slack for another take from phil/johnathan though.

Reply:
also, FWIW - the outputs of this convo will be out of scope for this ticket, we'll create a new one to adjust based on the feedback from the group

Reply:
cool, probably worht setting as a constant with //todo to address it when that changes

Reply:
note on min/max lease durations: can be set in seconds or milliseconds for dynamic changes (instead of years for example). ex if we say 1 year is the minumum, we put it in ms (31_536_000_000) instead of "1" so that if we change the minimum lease period's count we can leave the logic as-is.

Reply:
let's just allow years for now